src/main/java/oc/BuildaVater.java

```diff
@@ -194,8 +194,14 @@ public String getFormationType() {
 
     public int getCP() {
 
-        if (((String) kontingentBox.getSelectedItem()).equals("Super-Heavy Detachment") && (getCountFromInformationVector("Armiger selected") > 0)) {
+    	// Handle questor imperialis cp for super heavy detachement FAQ 
+    	//Page 106 – Knight Lances ability Change the last sentence to read: ‘The Command Benefit of each Imperial Knights Super-heavy Detachment 
+    	//is changed to ‘None’ if it does not contain at least one Imperial Knights Titanic unit, and is changed to ‘+6 Command Points’ if it contains at 
+    	//least three Imperial Knights Titanic units.’
+        if (((String) kontingentBox.getSelectedItem()).equals("Super-Heavy Detachment") && (getCountFromInformationVector("Armiger selected") == 3)) {
```

Comment:
- use `>` of `>=` comparison instead of `==`
- flip `equals` since otherwise it causes a `NullPointerException`
- remove redundant cast to `String`
- combine with second if block like this:
  ```java
  if (a && b) {
      ...
  } else if (a && c) {
      ...
  }
  ```
  to
  ```java
  if (a) {
      if(b) {
          ...
      } else if (c) {
          ...
      }
  }
  ```